clients/fides-js/src/lib/initOverlay.ts

```diff
@@ -9,6 +9,13 @@ import { ColorFormat, generateLighterColor } from "./style-utils";
 const FIDES_EMBED_CONTAINER_ID = "fides-embed-container";
 const FIDES_OVERLAY_DEFAULT_ID = "fides-overlay";
 
+/**
+ * Save a reference to the parent element used to render the overlay. This
+ * allows us to detect re-renders and defensively unmount previous versions of
+ * the overlay from the DOM if this occurs.
+ */
+let renderedParentElem: ContainerNode | undefined;
```

Comment:
This works! If we ever wanted to allow multiple instances of the widget on the same page, this would have to be per-instance, but baby steps 👍

Reply:
Yeah - to do that I'd prefer to have a better, more reliable overall `FidesContext` or something like that to actually store state in a clean way...